docs/OpenSSLSupport.md

```diff
@@ -148,6 +148,31 @@ Note that the code snippet for the RDRAND engine opt-in is required to use not o
 but also other OpenSSL APIs that internally depend on the RAND APIs. Alternatively, developers
 can implement their own RAND method to replace the default method via `RAND_set_rand_method` API.
 
+## Security Guidance of using OpenSSL APIs
+
+Few OpenSSL APIs allow users to configure an application to load files from the host filesystem.
+Such behaviors may have security implications given that the host filesystem is untrusted under the
+threat model of OE. For these reasons, OE recommends users not to use those APIs and
+put them into `deprecations.h` as part of OpenSSL headers that are shipped along with the release package.
+OE's default compiler settings explicitly include the header such that users get warnings when an
+enclave uses any of these APIs.
+
+See the following table for the detailed list of APIs.
+
+API | Original header | Comments
+:---:|:---:|:---|
+SSL_CTX_load_verify_locations | ssl.h | The API specifies the default locations from which an application looks up CA certificates for verification purposes. The API would allow an untrusted host to force the application to load fake certificates. |
```

Comment:
We should probably also provide guidance as to what the alternative that a developer should use is.